libcrux-ml-kem/src/lib.rs

```diff
@@ -12,21 +12,17 @@ pub(crate) mod constants;
 /// Helpers for verification and extraction
 mod helper;
 
-#[cfg(all(target_arch = "aarch64"))]
-mod intvec128;
-
 mod arithmetic;
 mod compress;
 mod constant_time_ops;
 mod hash_functions;
 mod ind_cpa;
-mod intvec;
-mod intvec32;
 mod matrix;
 mod ntt;
 mod polynomial;
 mod sampling;
 mod serialize;
+mod simd;
```

Comment:
This is not only simd, but polynomial really. It's fine to keep like this because it will need a refactor for runtime detection anyway. But the naming is a little off.